src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -40,6 +46,12 @@ VALIDATOR_HTTP_PORT_WAIT_DISABLED = None
 METRICS_PATH = "/metrics"
 VALIDATOR_SUFFIX_SERVICE_NAME = "validator"
 
+# The min/max CPU/memory that the validator node can use
+VALIDATOR_MIN_CPU = 50
+VALIDATOR_MAX_CPU = 300
```

Comment:
this is in milicore; so this is 0.3 cores. Is that okay?

Reply:
Yes, validator processes are very lightweight.